### PR TITLE
Add settings context panel

### PR DESCRIPTION
**Pull request by vishwam**

### CODE REVIEW

File: src/pages/App.tsx [hunk 1]
```diff
@@ -83,27 +87,52 @@ export class App extends React.Component<Properties, State>  {
...
-            label: 'Toggle theme',
-            onClick: this.handleThemeToggle
-          }
-        ]
-      },
-      attr: {
-        userMenuAriaLabel: 'User Menu',
-        mobileMenuAriaLabel: 'Application Menu'
+      branding: (
+        <>
```
Comment (vincenzocaruso) [inline src/pages/App.tsx]: what's it?

File: src/pages/App.tsx [hunk 2]
```diff
@@ -83,27 +87,52 @@ export class App extends React.Component<Properties, State>  {
...
+          <div onClick={this.handleSettingsToggle}>{loc('masthead')}</div>
+          {this.state.isSettingsExpanded && this.renderSettingsPanel()}
+        </>
+      ),
+      toolBarItems: {
+        settings: { title: 'settings', content: 'Settings', actions: { cancel: { event: undefined, label: 'cancel' } } },
```
Comment (vincenzocaruso) [inline src/pages/App.tsx]: should we localize the label?

File: src/pages/App.tsx (hunk 1 above)
Comment (vishwam) [reply src/pages/App.tsx]: It's a shorthand for React.Fragment, which allows you to return multiple children without wrapping in an array: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

File: src/pages/App.tsx (hunk 2 above)
Comment (vishwam) [reply src/pages/App.tsx]: added loc strings

**Pull request merged**